code/WorkInProgress/Cael_Aislinn/Rust/gyrotron.dm

```diff
@@ -72,6 +72,16 @@
 		</ul>
 	"}
 
+/obj/machinery/rust/gyrotron/canLink(var/obj/machinery/computer/rust_gyrotron_controller/object, var/link/context)
+	. = (istype(object) && get_dist(src, object) < RUST_GYROTRON_RANGE)
```

Comment:
Use return, it's much more readable.

Reply:
Once this is fixed I'll merge.  Tiny thing but it makes a difference.